m4_sessions/1804inning/winning_through_failure.md

```diff
@@ -8,11 +8,43 @@ Maddy Dannenberg
 Steven Lee Lawson
 
 # Name of Session: 
+Winning Through Failure --choosing the path of most resistance 
 
 ## Session Description (Why should students come to this session):
 
+Learn how to accept failure as being okay and is part of growing as a person.
+
 ## Session Outcomes (What will students learn/leave with?):
 
+How to be comfortable talking about your failures in a positive light as it's a popular interview question.
+Here's how I faced these challenges, and how I came through it.
+
 ## Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
+Intros: 15min
+  -Say your name and mod and a summer camp you went to and if you loved it or hated it, and why you chose this session.
+  
+
+  
+Stories of a few famous people of history (slides, video?) 4min
+Found a good video: [video](https://www.youtube.com/watch?v=zLYECIjmnQs)
+
+Slides: 
+  -US Grant 
+  -Jk Rowling 
+  -Elon Musk
+  -Steve Jobs 
+
+DISCLAIMER:
+*If this makes you feel uncomfortable, that's kind of the point as it's a commin interview question*
+
+In small groups: Talk about a time you have failed as small stories 
+  -start with groups of two, turn and talk
+  -then open up to table 
+  -then open it up to class to whoever wants to share
+
+Articles to read: 
+  [famous_folk](https://www.wanderlustworker.com/12-famous-people-who-failed-before-succeeding/)
+
+Social stigma of failure as a negative
```

Comment:
Say more of this...is this part of your closing?  How will you know students are walking away with the outcomes you want?